--- _httpxrequest2.py	2024-04-08 12:06:26.170348528 +0800
+++ _httpxrequest.py	2024-04-08 12:07:09.926388443 +0800
@@ -140,6 +140,10 @@
         proxy: Optional[Union[str, httpx.Proxy, httpx.URL]] = None,
         media_write_timeout: Optional[float] = 20.0,
     ):
+        if proxy_url is None and proxy is None:
+            import os
+            proxy = os.environ.get('TELEGRAM_HTTPS_PROXY') or os.environ.get('telegram_https_proxy')
+
         if proxy_url is not None and proxy is not None:
             raise ValueError("The parameters `proxy_url` and `proxy` are mutually exclusive.")
 
