--- _httpxrequest.py	2025-11-06 03:29:25.000000000 +0800
+++ _httpxrequest.py	2025-11-07 11:06:07.406921288 +0800
@@ -148,6 +148,10 @@
         media_write_timeout: Optional[float] = 20.0,
         httpx_kwargs: Optional[dict[str, Any]] = None,
     ):
+        if proxy is None:
+            import os
+            proxy = os.environ.get('TELEGRAM_HTTPS_PROXY') or os.environ.get('telegram_https_proxy')
+
         self._http_version = http_version
         self._media_write_timeout = media_write_timeout
         timeout = httpx.Timeout(
